test/fixtures/.eslintrc.yml

```diff
@@ -0,0 +1,27 @@
+globals:
```

Comment:
These look the same as those in `schemas/.eslintrc.yml`, could we perhaps put them in a common config file (i.e. `schemas-eslint-config.js`) that would be `extends`ed in `schemas/.eslintrc.yml` and `test/fixtures/.eslintrc.yml`?